integrated_channels/xapi/statements/learner_course_completion.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""
+X API Statement when learner completes a course.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from tincan import Activity, ActivityDefinition, Agent, Context, Extensions, LanguageMap, Result, Score, Verb
+
+from integrated_channels.xapi.statements.base import EnterpriseStatement
+
+
+class LearnerCourseCompletionStatement(EnterpriseStatement):
+    """
+    X-API Statement to serialize data related to course completion.
+    """
+
+    def __init__(self, user, course_overview, user_details, course_details, course_grade, *args, **kwargs):
+        """
+        Initialize and populate statement with learner info and course info.
+
+        Arguments:
+            user (User): Auth User object containing information about the learner enrolling in the course.
+            course_overview (CourseOverview): course overview object containing course details.
+            user_details (dict): A dict object containing learner info we want to send in X-API statement payload.
+            course_details (dict): A dict object containing course info we want to send in X-API statement payload.
+            course_grade (CourseGrade): User grade in the course.
+        """
+        kwargs.update(
```

Comment:
why use`kwargs` at all here if you're going to update each value through the methods below?

Reply:
base class `tincan.statement.Statement` expects these arguments in the init method that is why I updated the kwargs and passed the updated arguments.